File: Python/Creational/Singleton.py
class President:
    __instance = None

    @staticmethod
    def getInstance():
        if President.__instance == None:
            President()
        return President.__instance

    def __init__(self):
        if President.__instance != None:
            raise Exception("This class is a singleton!")
        else:
            President.__instance = self

if __name__ == '__main__':
    president1 = President.getInstance()
    president2 = President.getInstance()

if president1 is president2:
    print 'Objects are the same'
else:
    print 'Objects are not the same'

File: examples-per-language/php/creational/Singleton.php
<?php

use designPatternsForHumans\creational\Singleton\President;

require_once __DIR__ . '/autoload.php';

$president1 = President::getInstance();
$president2 = President::getInstance();

echo 'Are both presidents 100% identical? ';
var_dump($president1 === $president2);
